### PR TITLE
Yoink autodeploy from testauskoira-rs

### PR DESCRIPTION
**Pull request by RoyTakanen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,19 @@ to automatically set up the database, deleting any existing database.
...
 
+### Autodeploy
+
+This repository is configured to publish itself automatically to specified server. Clone this repository to the server (where you want the bot to run) and set correct values `.env` file. If there are new change to `main` branch, the server receives an automatic update of the new image (Github worker builds this image). 
+
+It is also important to specify the following keys in Github Secrets:
+
+| Key | Value |
+| --- | ----- |
+| SSH_DIR | Full path to cloned repository (e.g. `/home/testauskoira/testauskoira-rs`) |
```
Comment (lajp) [inline README.md]: /home/testauskoira/testauskoira-rs KEKW

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      with:
+        host: ${{ secrets.SSH_IP }}
+        username: ${{ secrets.SSH_USER }}
+        key: ${{ secrets.SSH_PRIVATE_KEY }}
+        command_timeout: 180m
+        script: |
+          cd ${{ secrets.SSH_DIR }}
+          git pull
+          docker-compose pull
+          docker-compose up -d
```
Comment (lajp) [inline .github/workflows/deploy.yml]: Add newline

**Review by lajp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RoyTakanen) [reply README.md]: Done

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (RoyTakanen) [reply .github/workflows/deploy.yml]: Done

**Review by lajp — APPROVED** [review]
LGTM now

**Comment by lajp** [comment]
I can merge this but now to the tricky part.. how does it get setup on the host

**Pull request merged**